code/game/machinery/Sleeper.dm

```diff
@@ -449,25 +453,23 @@
 /obj/machinery/sleeper/ex_act(severity)
 	if(filtering)
 		toggle_filter()
+	var/mob/M = src.occupant

```

Comment:
that makes sense, I should probably not try to do quick fixes when I'm tired in the future